components/clp-package-utils/clp_package_utils/scripts/compress.py

```
@@ -192,6 +172,15 @@ def main(argv):
         logger.exception("Failed to load config.")
         return -1
 
+    # Validate logs_input type is FS
+    if clp_config.logs_input.type != StorageType.FS:
+        logger.error(
+            f"Filesystem compression requires logs_input.type to be '{StorageType.FS}', "
+            f"but configured type is '{clp_config.logs_input.type}'. "
+            f"For S3 compression, use compress-from-s3.sh instead."
```

Ruff (0.14.1)

Logging statement uses f-string

(G004)